### PR TITLE
change all instances of 'story' to 'page'

### PR DESCRIPTION
**Pull request by christianp**
This commit replaces all instances of 'story' or 'stories' with 'page' or 'pages'.

The settings STORY_INDEX and COMMENTS_IN_STORIES are now PAGE_INDEX and COMMENTS_IN_PAGES

see getnikola/nikola#1891

### CODE REVIEW

File: docs/internals.txt [hunk 1]
```diff
@@ -88,14 +88,14 @@ one of the existing ones.
     You can do doctests, you can do unit tests, you can do integration tests. There is support
     for all of them.
 
-Posts and Stories
+Posts and Pages
 -----------------
```
Comment (Kwpolska) [inline docs/internals.txt]: Underline needs to match new title length.

File: docs/manual.txt [hunk 2]
```diff
@@ -815,26 +815,22 @@ There are multiple configuration variables dedicated to each of the three taxono
...
 
 The default configuration expects the page's metadata and text files to be on the
-``stories`` folder, but that can be changed (see ``PAGES`` option above).
+``pages`` folder, but that can be changed (see ``PAGES`` option above).
 
 You can create the page's files manually or use the ``new_post`` command
 with the ``-p`` option, which will place the files in the folder that
 has ``use_in_feed`` set to False.
 
-In some places (including default directories and templates), pages are called
```
Comment (Kwpolska) [inline docs/manual.txt]: I’d prefer this explainer to be left in.

File: docs/sphinx/creating-a-site.txt [hunk 3]
```diff
@@ -1 +1,185 @@
-../creating-a-site.txt
```
Comment (Kwpolska) [inline docs/sphinx/creating-a-site.txt]: Your git client seems to have broken symlinks.

Comment (christianp) [inline docs/sphinx/creating-a-site.txt]: I'm on Windows. How do I fix that?

Comment (Kwpolska) [inline docs/sphinx/creating-a-site.txt]: You can’t; we will if we go ahead with this.

**Comment by Kwpolska** [comment]
To do (by us):

- [ ] fix baseline
- [ ] fix symlinks
- [ ] offer settings migrations

@ralsina, mind reviewing this, too?

**Comment by Kwpolska** [comment]
@christianp Alright, I had some time to fix stuff and this now looks better. Please add an entry to CHANGES.txt and your name to AUTHORS.txt. You might also want to [see how it looks now](https://github.com/getnikola/nikola/pull/2518/files). Thanks for contributing to Nikola — we hope you’ll stay with us for longer!

I left `story.tmpl` as-is because this would break too much stuff. Perhaps in v8?

@ralsina (and anyone else): Reviews welcome.

**Comment by ralsina** [comment]
@Kwpolska sorry, I have absolutely no Nikola time for personal reasons until next week at least. Anything has my +1 I'd rather have master break a bit than letting contributions wither.

**Comment by Kwpolska** [comment]
@ralsina Okay, I’ll merge this now, but please look through the files changed whenever you have a while (no rush).

**Pull request merged**